techmatters/hrm
# CHI-1032: Handle missing 'name' property

Primary Reviewer: @robert-bo-davis 

## Description

* HRM searches won't error if ChildInformation or CallerInformation don't have a 'name' property on a contact
* Extracted function to infer name from a caller or child information blob to a separate function

### Checklist

### Related Issues
Fixes CHI-1032

### Verification steps

* Create contacts where no name is defined on the form
* Search for the contact created
* The search should return the contact with no name
